docker/Dockerfile-workers

```diff
@@ -26,7 +26,7 @@ RUN dnf -y install \
     && dnf clean all
 
 ENV OPM_SRC="/src"
-ENV OPM_TAG="v1.14.2"
+ENV OPM_TAG="v1.16.0"
```

Comment:
The one deployed in IIB is `v1.16.1`. Could you please update this?

Reply:
Are you sure? On the ansible playbooks repo:
```
$ files/opm version
Version: version.Version{OpmVersion:"v1.16.0", GitCommit:"eb9fff5", BuildDate:"2021-02-09T13:53:02Z", GoOs:"linux", GoArch:"amd64"}
```

Reply:
That is strange. I'm pretty sure I updated it to 1.16.1: https://gitlab.cee.redhat.com/ansible-playbooks/iib-playbooks/-/merge_requests/126